### PR TITLE
Mobile-Version-Skeleton

### PR DESCRIPTION
**Pull request by Hirwacardinalloic**
## Description
In this PR, we create a Mobile Version skeleton.

## General requirements
 - [x] No linter errors.
 - [x] Follow Github flow.
 - [x] Professional Documentation.
## project requirements:
 - [x] Added Mobile Version skeleton HTML and CSS
 
Happy reviewing 😃

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+            <h4 class="ico">LET’S CONNECT</h4>
+            <ul class="ico-img">
+              <li>
+                <a href="#"
+                  ><img
+                    class="t-social-media"
+                    src="./images/github.png"
+                    alt="twitter"
+                /></a>
+              </li>
```
Comment (TadesseDev) [inline index.html]: - [ ] Dear @Hirwacardinalloic, I'm impressed with what you have just done right now. One thing left to be addressed to be exact with the Figma design is that your design overflows on a `375px` screen size. Please make sure to add the margins (top, bottom, left, right) defined in the design. 

### Figma design 
background2

### Your design on `375px` screen
background

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Hirwacardinalloic 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great job addressing almost all the requirements.
I just have a single suggestion that needs to be addressed and you will be all good.
omg-oh

## Highlights
 - [ ] Good job overall following the template design 💯
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  Good commit history 🥇
 - [ ]  Good PR title/description 💯
 - [ ]  All linters passed 🥇
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+
+.box-contain ul {
+  padding: 0;
+  margin: 0;
+}
+
+.ptflio-item {
+  width: 90%;
+  margin: 0 auto;
+  list-style: none;
```
Comment (Owusu-Desmond) [inline style.css:105]: - [ ] **As requested by the previous reviewer**, Please kindly make sure your project is responsive on mobile screens. But your project has a horizontal scroll on the `375px` screen size. Think you should just the styling of your cards and containers to make them responsive to remove the horizontal scroll. 
  **You can see the screenshot below**. :pray: 
  Screenshot from 2022-12-03 21-41-17

  **You can see from this too**

  [Screencast from 12-03-2022 09:34:12 PM.webm](https://user-images.githubusercontent.com/92303891/205463225-c6b4b23c-7e55-47d3-a06f-48b3fd07502c.webm)

File: style.css [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+.butto {
+  font-size: 19px;
+  color: #396df2;
+  padding: 13px;
+  background-color: transparent;
+  margin-top: 2rem;
+  border: 1px solid #396df2;
+  border-radius: 6px;
+}
+
```
Comment (Owusu-Desmond) [inline style.css]: - [OPTIONAL ] Although the previous reviewer did not see this but please make the  `See project`  `button` follow the button styling: Active, Hover, and Pressed (see design)
Screenshot from 2022-12-03 19-12-27

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Hirwacardinalloic  :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Hirwacardinalloic** [comment]
hello, am trying to commit a new change but the pull request is not showing
me the latest commits, what happened?
any help?

File: style.css (hunk 3 above)
Comment (Debas-31) [reply style.css]: - [OPTIONAL] - Please kindly implement the previous code reviewer's suggestion, even it's optional, about buttons effects such as `Enabled`, `Hover`, `Pressed`, and `Disabled`

**Review by Debas-31 — APPROVED** [review]
> ## Hi @Hirwacardinalloic  :wave:, 

# Status: Approved :heavy_check_mark: 

Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- No linters error :heavy_check_mark:
- PR title and summary :heavy_check_mark: 
- Commit message :heavy_check_mark:
- Figma template usage :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**